index.html

```diff
@@ -0,0 +1,172 @@
+                <li><a class="top-a" href="https://www.twitter.com/"><img src="./images/ic_twitter.svg"></a></li>
+                <li><a class="top-a" href="https://www.instagram.com/"><img src="./images/ic_linkedin.svg"></a></li>
+                <li><a class="top-a" href="#join">English</a></li>
+                <li><a class="top-a" href="#sponsor">My page</a></li>
+                <li><a class="top-a" href="#news">Logout</a></li>
+            </ul>
+
+        </nav>
+
+        <nav class="nav_menu">
+
+            <img class="logo" src="./images/logo.PNG" alt="logo">
+            <ul class="desk-bar">
+                <li><a class="top-b" href="./">Home</a></li>
+                <li><a class="top-b" href="./about.html">About</a></li>
+                <li><a class="top-b" href="#join">Join</a></li>
+                <li><a class="top-b" href="#sponsor">Sponsor</a></li>
+                <li><a class="top-b" href="#news">News</a></li>
+            </ul>
+        </nav>
+
+    </header>
+
+
+    <main>
+
+        <section class="hidemobile" id="mobile_menu">
+            <a class="topx" href="#"><img src="./images/xtopmenu.svg" alt=""></a>
+
+            <ul class="top_menu">
+                <li><a class="top-a1" href="./">Home</a></li>
+                <li><a class="top-a2" href="./about.html">About</a></li>
+                <li><a class="top-a3" href="#join">Join</a></li>
+                <li><a class="top-a4" href="#sponsor">Sponsor</a></li>
+                <li><a class="top-a5" href="#news">News</a></li>
+            </ul>
+
+        </section>
+
+        <div id="content" class="showcontent">
+            <div class="background">
+                <section id="home">
+                    <span class="hamb"><img src="./images/bx-menu.svg" alt="hamb icon"></span>
+                    <span class="span1">"Hello! Trading Community"</span>
+                    <h1 class="title1">LATIN AMERICA TRADING CONFERENCE 2023</h1>
+                    <p class="home-p">An exciting event sharing with profitable traders with more than 10 years of
+                        experience operating the
+                        financial markets.</p>
+                    <h2>2023.06.20(TUE) ≈ 22(THU)</h2>
+                    <span class="span2"> @ International Business and Exhibition Center of Bogotá, Colombia -
+                        Corferias</span>
+
+                </section>
+            </div>
+
+            <section class="mainprogram">
+
+                <h3 class="mp">Main program</h3>
+                <hr class="division_mh3">
+
+                <div class="mainpdesk">
+                    <div class="p1">
+                        <div class="logo1"><img src="./images/program_icon_01.png" alt=""></div>
+                        <h4 class="h41">Lecture</h4>
+                        <p class="p1p1">Listen to the speakers from various countries about their experience at
+                            financial
+                            markets.</p>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add hover effect to `Main Program` cards like a [Behance design ](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏 

![image](https://user-images.githubusercontent.com/63915024/203848085-88eb8dbb-adba-4c86-8a1c-64de239eea9e.png)